index.js

```diff
@@ -3,16 +3,23 @@ new Vue({
   data: {
   	todos: [
   		{ text: 'test', done: false, id: Date.now() }
-  	]
+  	], 
+    activeTodos: [], 
+    completedTodos: []
   },
   methods: {
   	addTodo({target}){
     	this.todos.push({text: target.value, done: false, id: Date.now()})
       target.value = ''
+      console.log(this.todos)
     },
     removeTodo(id) {
-    	console.log('removeTodo has been clicked')
     	this.todos = this.todos.filter(todo => todo.id !== id)
-    }
+    },
+    updateActiveAndCompletedLists() {
+      this.completedTodos = this.todos.filter(todo => todo.done == true)
+      this.activeTodos = this.todos.filter(todo => todo.done == false)
+      this.todos = this.completedTodos
+      }

```

Comment:
create a new function here called `showActive()` and inside it, set the variable `showActive` to be true and set the variable `showCompleted` to be false.

create another new function here called `showCompleted()` and inside it, set the variable `showCompleted` to be true and the variable `showActive` to be false.